go/vt/wrangler/testlib/backup_test.go

```diff
@@ -684,8 +702,10 @@ func TestDisableActiveReparents(t *testing.T) {
 			},
 		},
 	}
+	sourceTablet.FakeMysqlDaemon.SetReplicationSourceInputs = []string{fmt.Sprintf("%s:%d", primary.Tablet.MysqlHostname, primary.Tablet.MysqlPort)}
 	sourceTablet.FakeMysqlDaemon.ExpectedExecuteSuperQueryList = []string{
 		"STOP SLAVE",
+		"FAKE SET MASTER",
```

Comment:
This looks wrong.. when `DisableActiveReparents` is set, we _don't_ want to start replication.
What am I missing?

Reply:
We aren't starting replication, we are just setting the correct primary

Reply:
Let me rephrase. We should not touch replication in _ANY_ way when this flag is set.

Reply:
Fixed